### DISCUSSION TITLE
Why did you remove move-attr-prop.js?

### DISCUSSION THREAD

**Discussion by uploadersoasis** [uBlockO/uBO-Scriptlets#27] [Q&A] [answered]
Why did you remove the "move-attr-prop" scriptlet in commit 7224cfe?  I use that in my rules.  I can put that scriptlet in my own scriptlet file which supplements yours, but I would like to know why you removed it in the first place.

**Comment by uBlock-user** [comment]
Because that was added for multiup.org website. gorhill added a dedicated scriptlet for that site, so I removed it. 

I remove many scriptlets which once get added to uBO itself. 

>  I use that in my rules.

Whats your use-case ?

**Comment by uploadersoasis** [comment]
> Whats your use-case ?

It was for multiup primarily, but I also use it for another file host, 4shared.

Because that scriptlet is powerful and handy, I don't see it as just site-specific.

My concern in asking was to know if it was for security or compatibility reasons before I added it back.

**Reply by uBlock-user** [reply]
https://github.com/gorhill/uBlock/commit/d3b95c1c682d4ee5c592f08c1e7372dbd8dfd9c2

Will that scriptlet work for you ?

**Reply by uBlock-user** [reply]
Please answer the above question as asked, so I can decide whether I should add the scriptlet back or not.

**Comment by uploadersoasis** [comment]
> [gorhill/uBlock@d3b95c1](https://github.com/gorhill/uBlock/commit/d3b95c1c682d4ee5c592f08c1e7372dbd8dfd9c2)
> 
> Will that scriptlet work for you ?

I'll have to test it.  I had found that only 1fichier links @ multiup wouldn't work using my general multiup rules using the saa and set-attr scriptlets to change the "link" and "formaction" values of the existing download buttons.

Of course that multiup scriptlet won't work for my 4shared usage.  I can add move-attr-prop to my custom scriptlet file if you don't want to re-add it.

**Reply by uBlock-user** [reply]
For multiup, that scriptlet would work with the above filter - `multiup.*##+js(multiup)`, try the same for `4shared` too.

**Reply by uBlock-user** [reply, marked as answer]
> Of course that multiup scriptlet won't work for my 4shared usage. I can add move-attr-prop to my custom scriptlet if you don't want to re-add it.

Okay, I will add it back.

https://github.com/uBlock-user/uBO-Scriptlets/commit/070d203170a8eaed8a662ed1c1f6081e6fa00bac
